_posts/2019/2019-09-18-you.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title: "ePages YOU 19 - from a perspective of two newbies"
+date: '2019-09-18'
+header_image: private/you2019.jpg
+header_position: bottom
+header_overlay: true
+category: events
+tags: ["company", "events", "workshops", "company culture"]
+authors: ["Anna-Lena", "Mario"]
+about_authors: ["alanghans", "mkoester"]
+---
+
+## Day one
+
+### It’s all about you
+
+This year’s YOU agenda offered many very interesting presentations of different departments of ePages, a cool ice breaker, workshops, and an exciting panel discussion with four online merchants.
+And, of course, there was also a fun team event planned.
+All in all a lot to look forward to!
+
+Coming to the office last Thursday was a bit different than usual.
+Many people were already standing together, talking and laughing in the hallway.
+This first impression was already promising a welcoming and more casual atmosphere.
+Quickly grabbing a snack, it was time to join the others.
+
+### "Welcome to the ePages YOU!"
+
+Wilfried Beeck, CEO of ePages, gave us an update on where ePages is heading to in the future.
+He also presented how ePages is working across the world with existing and future partners.
+
+{% twitter https://twitter.com/epagesdevs/status/1172044512421658630 %}
+
+Afterwards, Tatu Wikman, CTO of the [Vilkas Group](https://www.vilkasgroup.com/), who has been an ePages’ technology partner for many years now, joined us and talked about how Vilkas works and what they offer to achieve the best possible ePages shop solution.
```

Comment:
ePages' > ePages

Reply:
This is already difficult. It read as if we're not delivering the best solution already and someone outside takes care of this. Would rephrase this.